src/orquestra/quantum/backends/symbolic_backend.py

```diff
@@ -4,10 +4,11 @@
 from orquestra.quantum.api.backend import QuantumBackend
 from orquestra.quantum.circuits import Circuit
 from orquestra.quantum.measurements import Measurements
-from orquestra.quantum.symbolic_simulator import SymbolicSimulator
 
+from .symbolic_simulator import SymbolicSimulator
 
-class MockQuantumBackend(QuantumBackend):
+
+class SymbolicBackend(QuantumBackend):
```

Comment:
I would actually argue against this change.
I think that having the name `MockQuantumBackend` is much clearer that this is something that's only for test purposes.
While with `SymbolicBackend` it's looks that this might be something a person could potentially want to use.
Why changing it?